android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapFragment.java

```diff
@@ -676,23 +675,20 @@ public void showMapNotAvailableMessage(NoMapAvailableMessage noMapAvailableMessa
 
     public void showPausedDueToNoMotionMessage(boolean show) {
         mRootView.findViewById(R.id.scanning_paused_message).setVisibility(show ? View.VISIBLE : View.INVISIBLE);
-        if (mMapLocationListener != null) {
-            mMapLocationListener.pauseGpsUpdates(show);
-        }
         updateGPSInfo(0, 0);
         dimToolbar();
     }
 
     public void start() {
         if (mMapLocationListener != null) {
-            mMapLocationListener.setActiveLocationUpdatesEnabled(true);
+            mMapLocationListener.setMapVisible(true);
```

Comment:
The map becomes visible in `MapFragment.doOnResume` and invisible in `MapFragment.onPause`. These methods indicate a scanner state change while the map is visible.